src/routes/accounts/counterfactual-safes/counterfactual-safes.controller.spec.ts

```diff
@@ -584,4 +584,337 @@ describe('CounterfactualSafesController', () => {
       ).not.toHaveBeenCalled();
     });
   });
+
+  describe('Delete Counterfactual Safe', () => {
+    it('should delete a Counterfactual Safe', async () => {
```

Comment:
We should add test coverage regarding a datasource error. These assume that deletion was successful.

Reply:
Added in [650d3f6](https://github.com/safe-global/safe-client-gateway/pull/1795/commits/650d3f6adc010330d6a12092afbd3a09e3df7c80) (I've added that test also to  the rest of endpoints)